src/index.rs

```diff
@@ -32,6 +34,18 @@ pub mod symbol;
 mod tests;
 pub mod visitor;
 
+/// Type alias for a HashMap using the `fx` hashing algorithm, see https://github.com/rust-lang/rustc-hash
+pub type FxHashMap<K, V> = HashMap<K, V, BuildHasherDefault<FxHasher>>;
+
+/// Type alias for a HashSet using the `fx` hashing algorithm, see https://github.com/rust-lang/rustc-hash
+pub type FxHashSet<K> = HashSet<K, BuildHasherDefault<FxHasher>>;
+
+/// Type alias for an IndexMap using the `fx` hashing algorithm, see https://github.com/rust-lang/rustc-hash
+pub type FxIndexMap<K, V> = IndexMap<K, V, BuildHasherDefault<FxHasher>>;
+
+/// Type alias for a IndexSet using the `fx` hashing algorithm, see https://github.com/rust-lang/rustc-hash
+pub type FxIndexSet<K> = IndexSet<K, BuildHasherDefault<FxHasher>>;
+
```

Comment:
Are we ok with where these type-aliases are defined at? Would it be better to define them in another crate such as (the already existing) `plc_util` so that every other crate can make use of this without importing `plc`?